Modules/xASL_module_BIDS2Legacy.m

```diff
@@ -44,6 +44,9 @@
     
     % Print feedback
     xASL_adm_BreakString('BIDS to ExploreASL LEGACY CONVERSION');
+	
+	% Update paths
+	x = xASL_imp_UpdateDatasetRoot(x);
```

Comment:
OK, this function already existed, and is now removed from the import module?

Reply:
I've kept it in the import module not to mess up defacing.